crates/lovely-core/src/sys.rs

```diff
@@ -28,6 +28,14 @@ pub static LUA_LIB: Lazy<Library> = Lazy::new(|| {
 });
 
 
+#[cfg(target_os = "linux")]
+pub static LUA_LIB: Lazy<Library> = Lazy::new(|| {
+    unsafe {
+        Library::new("liblua5.1.so").unwrap()
+    }
+});
```

Comment:
this from the other linux PR? probably get rid of this

Reply:
Okay, did that